test/IntegrationTests/GraphQLNativeTests.cs

```diff
@@ -0,0 +1,43 @@
+// <copyright file="GraphQLNativeTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#if NET6_0_OR_GREATER
+
+using Xunit.Abstractions;
+
+namespace IntegrationTests;
+
+public class GraphQLNativeTests : GraphQLTests
```

Comment:
Similar to other suggested renames:
```suggestion
public class GraphQLNativelyInstrumented : GraphQLTests
```